packages/api-explorer/package.json

```diff
@@ -4,14 +4,15 @@
   "version": "6.1.1",
   "main": "dist/index.js",
   "dependencies": {
-    "@json-editor/json-editor": "^1.4.0-beta.0",
+    "@fortawesome/fontawesome-free": "^5.13.0",
+    "@json-editor/json-editor": "2.2.1",
     "@mia-platform/markdown": "^6.1.0",
     "@mia-platform/oas-extensions": "^6.1.0",
     "@mia-platform/syntax-highlighter": "^6.1.0",
     "@mia-platform/variable": "^6.1.0",
     "choices.js": "^9.0.1",
     "classnames": "^2.2.5",
-    "fetch-har": "^2.0.0",
+    "fetch-har": "2.0.0",
```

Comment:
There are some security issues resolved by the updated version. Why it is fixed?

Reply:
We updated to 2.2.0 and works.
If update to 2.2.1 fails due to this commit: https://github.com/readmeio/fetch-har/commit/032d787

Headers of "Request API" seems not to be supported. (tested also with node 12)

Reply:
Ok with 2.2.0!